packages/cashier/src/Stores/__tests__/crypto-fiat-converter-store.spec.js

```diff
@@ -0,0 +1,323 @@
+import CryptoFiatConverterStore from '../crypto-fiat-converter-store';
+
+let crypto_fiat_converter_store, root_store, WS;
+
+beforeEach(() => {
+    WS = {
+        send: jest.fn(),
+    };
+    root_store = {
+        modules: {
+            cashier: {
+                account_transfer: {
+                    container: 'account_transfer',
+                    setAccountTransferAmount: jest.fn(),
+                    validateTransferFromAmount: jest.fn(),
+                    validateTransferToAmount: jest.fn(),
+                },
+                general_store: {
+                    active_container: 'deposit',
+                    calculatePercentage: jest.fn(),
+                    percentageSelectorSelectionStatus: jest.fn(),
+                },
+                withdraw: {
+                    validateWithdrawFromAmount: jest.fn(),
+                    validateWithdrawToAmount: jest.fn(),
+                },
+            },
+        },
+    };
+    crypto_fiat_converter_store = new CryptoFiatConverterStore({ WS, root_store });
+});
+
+describe('CryptoFiatConverterStore', () => {
+    it('should set converter_from_amount to the given amount', () => {
+        crypto_fiat_converter_store.setConverterFromAmount(100);
+
+        expect(crypto_fiat_converter_store.converter_from_amount).toBe(100);
+    });
+
+    it('should set converter_to_amount to the given amount', () => {
+        crypto_fiat_converter_store.setConverterToAmount(100);
+
+        expect(crypto_fiat_converter_store.converter_to_amount).toBe(100);
+    });
+
+    it('should set error to converter_from_error', () => {
+        const error = new Error('Something went wrong');
+
+        crypto_fiat_converter_store.setConverterFromError(error);
+
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual(error);
+    });
+
+    it('should set error to converter_to_error', () => {
+        const error = new Error('Something went wrong');
+
+        crypto_fiat_converter_store.setConverterToError(error);
+
+        expect(crypto_fiat_converter_store.converter_to_error).toEqual(error);
+    });
+
+    it('should set is_timer_visible to a truthy value', () => {
+        const error = new Error('Something went wrong');
+
+        crypto_fiat_converter_store.setIsTimerVisible(true);
+
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeTruthy();
+    });
+
+    it('should set is_timer_visible to a false value', () => {
+        const error = new Error('Something went wrong');
+
+        crypto_fiat_converter_store.resetTimer(false);
+
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeFalse();
+    });
+
+    it('should set is_timer_visible to a false value', () => {
+        const error = new Error('Something went wrong');
```

Comment:
@vinu-deriv believe that we don't need this line, because we don't use error variable in this test case